resources/frontend_shared/static_viz_interface.js

```diff
@@ -23,11 +23,12 @@ function combo_chart(series, settings, colors) {
   });
 }
 
-function timeseries_waterfall(data, labels, settings, instanceColors) {
-  return StaticViz.RenderChart("timeseries/waterfall", {
+function waterfall(data, labels, settings, waterfallType, instanceColors) {
```

Comment:
Change the BE interface to have just a single `waterfall` function, since FE also uses a single component for rendering both time-series and categorical waterfall charts now.